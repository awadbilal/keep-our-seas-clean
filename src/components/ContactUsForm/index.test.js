import React from "react";
import renderer from "react-test-renderer";
import ContactUsForm from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<ContactUsForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
