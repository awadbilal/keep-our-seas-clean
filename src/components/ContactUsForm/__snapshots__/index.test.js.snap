// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main pt-5 pb-5 fdb-block container-fluid"
>
  <div
    className="align-items-center row"
  >
    <div
      className="px-4 col-lg-3 col-md-12 col-sm-12 offset-lg-1"
    >
      <h2
        className="title"
      >
        contact.getInTouch.title
      </h2>
      <p
        className="mt-4"
      >
        contact.getInTouch.paragraph
      </p>
    </div>
    <div
      className="px-4 col-lg-7 col-md-12 col-sm-12"
    >
      <div
        className="row row-cols-md-12 row-cols-sm-12"
      >
        <form
          className=""
        >
          <div
            className="mb-3 row"
          >
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridName"
              >
                form.name
              </label>
              <input
                className="form-control"
                id="formGridName"
                name="name"
                onChange={[Function]}
                placeholder="formPlaceholder.name"
                required={true}
                type="text"
                value=""
              />
            </div>
            <div
              className="col"
            >
              <label
                className="form-label"
                htmlFor="formGridSurname"
              >
                form.surname
              </label>
              <input
                className="form-control"
                id="formGridSurname"
                name="surname"
                onChange={[Function]}
                placeholder="formPlaceholder.surname"
                required={true}
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            className="mb-3"
          >
            <label
              className="form-label"
              htmlFor="formGridEmail"
            >
              form.email
            </label>
            <input
              className="form-control"
              id="formGridEmail"
              name="email"
              onChange={[Function]}
              placeholder="formPlaceholder.email"
              required={true}
              type="email"
              value=""
            />
          </div>
          <div
            className="mb-3"
          >
            <label
              className="form-label"
              htmlFor="exampleForm.ControlTextarea1"
            >
              form.message
            </label>
            <textarea
              className="form-control"
              id="exampleForm.ControlTextarea1"
              name="message"
              onChange={[Function]}
              placeholder="formPlaceholder.message"
              required={true}
              rows={3}
              value=""
            />
          </div>
          <button
            className="submitButton btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            form.submit
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
